### ISSUE TITLE
cifar10 example fails with asegmentation fault

### ISSUE THREAD

**Issue by eduj36** [tensorflow/models#1084]
cifar10 example fails with a segmentation fault as below.

    ubuntu@ubuntu:~/work/tensorflow/models2/tutorials/image/cifar10$ python cifar10_train.py --max_steps=10 --batch_size=4
    I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcublas.so.8.0 locally
    I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcudnn.so.5 locally
    I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcufft.so.8.0 locally
    I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcuda.so.1 locally
    I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcurand.so.8.0 locally
    Filling queue with 20000 CIFAR images before starting to train. This will take a few minutes.
    W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE3 instructions, but these are available on your machine and could speed up CPU computations.
    W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on your machine and could speed up CPU computations.
    W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
    W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
    W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
    W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
    I tensorflow/core/common_runtime/gpu/gpu_device.cc:885] Found device 0 with properties:
    name: GeForce GTX TITAN X
    major: 5 minor: 2 memoryClockRate (GHz) 1.076
    pciBusID 0000:03:00.0
    Total memory: 11.92GiB
    Free memory: 11.81GiB
    I tensorflow/core/common_runtime/gpu/gpu_device.cc:906] DMA: 0
    I tensorflow/core/common_runtime/gpu/gpu_device.cc:916] 0:   Y
    I tensorflow/core/common_runtime/gpu/gpu_device.cc:975] Creating TensorFlow device (/gpu:0) -> (device: 0, name: GeForce GTX TITAN X, pci bus id: 0000:03:00.0)
    2017-03-01 10:40:00.056918: step 0, loss = 4.68 (0.4 examples/sec; 10.139 sec/batch)
    Segmentation fault (core dumped)

system info:
ubuntu 14.04.2
tensorflow-gpu 1.0.0 installed using pip
cuda 8.0
cudnn 5.1.5

I am pretty new to tensorflow and python. Can you advise me how to fix or debug this?

**Comment by eduj36** [comment]
found that model/slim also fails.

    ubuntu@ubuntu:~/work/tensorflow/model/slim$ python train_image_classifier.py --train_dir=./train/inceptionv3/flowers --dataset_name=flowers --dataset_split_name=train --dataset_dir=./data/flowers --log_every_n_steps=10 --trace_every_n_steps=10 --max_number_of_steps=50 --batch_size=32
    I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcublas.so.8.0 locally
    I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcudnn.so.5 locally
    I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcufft.so.8.0 locally
    I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcuda.so.1 locally
    I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcurand.so.8.0 locally
    *** Error in `python': double free or corruption (!prev): 0x0000000000e544b0 ***
    Aborted (core dumped)

**Comment by eduj36** [comment]
Downgrading tensorflow to 0.12 solved this issue.
